### ISSUE TITLE
Not detecting hanging closing tags

### ISSUE THREAD

**Issue by tobias-edwards** [yeonjuan/html-eslint#171]
The plugin doesn't seem to be picking up on hanging closing tags e.g.

```html
</p>
```

When included in a large HTML file, the HTML is then invalid and formatters like Prettier are unable to execute.

Package versions:
```json
{
    "@html-eslint/eslint-plugin": "^0.22.0",
    "@html-eslint/parser": "^0.22.0",
    "eslint": "^8.56.0",
}
```

I tried using `@html-eslint/require-closing-tags` but it also does not pick up on hanging closing tags.

**Comment by yeonjuan** [comment]
Hi @tobias-edwards Thank you for the report. Are you suggesting adding a lint rule that checks for hanging closing tags?
